taichi/rhi/device.h

```diff
@@ -148,6 +148,13 @@ class ResourceBinder {
     TI_NOT_IMPLEMENTED
   }
 
+  virtual void rw_image(uint32_t set,
+                        uint32_t binding,
+                        DeviceAllocation alloc,
+                        int lod) {
+    TI_NOT_IMPLEMENTED
```

Comment:
This can be `=0` since you have overriden each device impl.

Reply:
This is intended for devices that do not support this doesn't need to have a override functions